crates/astria-conductor/tests/blackbox/helpers/macros.rs

```diff
@@ -15,15 +15,15 @@ macro_rules! block {
 
 #[macro_export]
 macro_rules! celestia_network_head {
-    (height: $height:expr) => {
+    (height: $height:expr,chain_id: $chain_id:expr $(,)?) => {
```

Comment:
Reduce noise by providing a pattern where `chain_id` is left unset, i.e. something like this:

```rust
macro_rules! celestia_network_head {
    (height: $height:expr) => {
        celestia_network_head!(height: $height, chain_id: crate::CELESTIA_CHAIN_ID,);
    }
    (height: $height:expr, chain_id: $expr) => {
        todo!("the actual definition");
    }
}
```